.github/workflows/flow-linux-platforms.yml

```diff
@@ -59,34 +64,83 @@ on:
         description: 'Whether to run standalone tests'
         type: boolean
         default: true
+      redis-ref:
+        type: string
 
 jobs:
   get-required-envs:
     runs-on: ubuntu-latest
     outputs:
-      platform: ${{ steps.get_platforms.outputs.platform }}
+      platforms_arm: ${{ steps.get_platforms.outputs.arm-platforms }}
+      platforms_x86: ${{ steps.get_platforms.outputs.x86-platforms }}
+      include_arm: ${{ steps.get_platforms.outputs.arm-include }}
+      include_x86: ${{ steps.get_platforms.outputs.x86-include }}
     steps:
       - name: Get platform
         id: get_platforms
+        shell: python
         run: |
-          if [ "${{ inputs.platform }}" == "all" ]; then
-            echo "platform=${{ env.ALL_IMAGES }}" >> $GITHUB_OUTPUT
-          else
-            echo "platform=['${{ inputs.platform }}']" >> $GITHUB_OUTPUT
-          fi
+          import os
+
+          requested_platform = '${{ inputs.platform }}'
+          requested_arch = '${{ inputs.architecture }}'
+
+          # Get the platforms to build on (set `x86-platforms` and `arm-platforms`)
+          if requested_platform == 'all':
+            x86-platforms = ${{ fromJson(env.ALL_X86_IMAGES) }} if requested_arch == 'all' or requested_arch == 'x86_64' else []
+            arm-platforms = ${{ fromJson(env.ALL_ARM_IMAGES) }} if requested_arch == 'all' or requested_arch == 'arm64'  else []
```

Comment:
The zip filename and in the `module.json` file we now use `aarch64` for arm. Consider use it here as well see we'll have some standart